hs_access_control/admin.py

```diff
@@ -0,0 +1,5 @@
+from mezzanine.pages.admin import PageAdmin
+from django.contrib.gis import admin
+from .models import *
+
+# admin.site.register(MyResource, PageAdmin)
```

Comment:
Module is not necessary *or* you should register some of the new models here, but either way not blank